src/databricks/labs/remorph/snow/experimental.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from typing import ClassVar
+
+from sqlglot import expressions as exp
+
+from databricks.labs.remorph.snow import databricks
+
+logger = logging.getLogger(__name__)
+
+
+class Databricks(databricks.Databricks):
```

Comment:
```suggestion
class DatabricksExperimental(databricks.Databricks):
```
otherwise we'll get import confusion bugs

Reply:
Done